label/search.go

```diff
@@ -36,8 +37,13 @@ type SearchCursor struct {
 }
 
 var searchTemplate = template.Must(template.New("search").Parse(`
-	SELECT{{if .UniqueKeys}} distinct on (lower(key)){{end}}
-		key, value, tgt_service_id
+	SELECT DISTINCT ON
+		{{if .UniqueKeys}}
+			(lower(key))
```

Comment:
We shouldn't use lower() in distinct, because keys (and values) are case-sensitive.

Reply:
That's what we had before:
```
SELECT{{if .UniqueKeys}} distinct on (lower(key)){{end}}
		key, value, tgt_service_id
```
I had just broken it up into lines to tweak the rest. What would you suggest we use instead, just `key`?